flow/connectors/eventhub/hubmanager.go

```diff
@@ -104,6 +104,7 @@ func (m *EventHubManager) closeProducerClient(ctx context.Context, pc *azeventhu
 
 func (m *EventHubManager) Close(ctx context.Context) error {
 	var allErrors error
+	var hubsClosed int
```

Comment:
@serprex I've noticed the close of a single hub taking over 30 seconds, should we do a heart beat routine here?

Reply:
wow, yeah